src/routes/AccountDetailsContainer.js

```diff
@@ -39,6 +41,11 @@ class AccountDetailsContainer extends React.Component {
     activeRecord: {
       accountId: '0',
     },
+    loans: {
+      type: 'okapi',
+      records: 'loans',
+      path: 'circulation/loans?query=(userId=:{id})&limit=100',
```

Comment:
As far as I understand the idea, this solution requests the information about the first 100 loans and filter them to find the needed value. If there are 101 loans and user selects the item 101, it seems the application will work incorrectly. As far as I remember, the similar issue was reproducible a lot of times in scope of the ui-users. Maybe, it should be fixed in scope of the separated stories.

Reply:
Hmm, I do not know if there are any limitations (how many loans can have single user), but in current request we can just remove the part of query with limit, right? And if we cannot remove limit then should this problem be solved on BE side?

Reply:
If the limit=100 is removed, the server will use the value 10 by default.
In the best case, we should endpoint (or find other mechanisms) to get information about the target loan by id instead of requesting a lot of items. It seems, there are no accurate numbers about the amount of the possible loans per user. But according to the created Jira tickets (For example, UIU-1299), we should support more than 100-500 loans. Usually, we just increase the limit value keeping in mind, that we have to request hundreds of loans just to select one (It seems, this is not about a good performance). I think we can try to increase the limit value for now (temp solution) and discuss with PO new endpoint creation (if there are not any other ways to request the information about 1 loan) to handle this problem in the future.

My comment is not a change request. I just wanted to mention the possible issue.